modules/viewing-the-compliance-scan-status.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * manage-compliance/managing-compliance-20.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-the-compliance-scan-status_{context}"]
+= Viewing the compliance scan status
+
+By viewing the status of a compliance scan, you can efficiently monitor and analyze the health of your clusters.
```

Comment:
We should probably note that it may take a few minutes for results to be returned while the compliance operator executes the scans.